lib/helper/helper_functions.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+class HelperFunctions {
+  static String sharedPreferenceUserLoggedInKey = "ISLOGGEDIN";
+  static String sharedPreferenceUserNameKey = "USERNAMEKEY";
+  static String sharedPreferenceUserEmailKey = "USEREMAILKEY";
+
+  static Future<bool> saveUserLoggedInSharedPreference(
+      bool isUserLoggedIn) async {
+    SharedPreferences preferences = await SharedPreferences.getInstance();
```

Comment:
you can make it simplier and get instance of sharedPreferences on loading HelperFunction class and it would be like
`SharedPreferences preferences = await SharedPreferences.getInstance()` and it should be in class body. and after it all your functions could be transformed to arrow functions, because preferences would be already initialized